pkg/pipeline/ingest_decode/ingest_collector.go

```diff
@@ -109,7 +108,7 @@ func (w *TransportWrapper) Send(_, data []byte) error {
 }
 
 // Ingest ingests entries from a network collector using goflow2 library (https://github.com/netsampler/goflow2)
-func (ingestC *ingestCollector) Ingest(out chan<- []interface{}) {
+func (ingestC *ingestCollector) Ingest(out chan<- []config.GenericMap) {
 	ctx := context.Background()
```

Comment:
Should we change the name of the interface function (Ingest) so we don't confuse it with an Ingestor? Or do we now dispose with the distinction between stage names, and any first stage is called Ingest?

Reply:
sure, done